src/main/java/fr/insee/pogues/transforms/visualize/ModelMapper.java

```diff
@@ -0,0 +1,34 @@
+package fr.insee.pogues.transforms.visualize;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+public class ModelMapper {
```

Comment:
The name ModelMapper confuses me, maybe something like IOStreamsUtils?

Since it is a utility class that has no need to be instanciated, constructor can me made private.

Also the class feels a bit lost among the visualize package, maybe move it to a "utils" package somewhere.

Reply:
You're write